### PR TITLE
Ignite-6083 Null value has appeared in entry processor, but entry exists

### PR DESCRIPTION
**Pull request by voipp**

### CODE REVIEW

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+                .setMaxConcurrentAsyncOperations(0)
+                .setBackups(0));
+    }
+
+    /**
+     * Test for sequential entry processor invoking not null value.
+     */
+    public void testSequentialCallOnOneNode() throws Exception {
+        Ignite ignite = startGrid(0);
+        try {
```
Comment (SomeFire) [inline modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: Need empty line,

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        finally {
+            stopAllGrids();
+        }
+    }
+
+    /**
+     * Cache entry processor checking whether entry has got non-null value.
+     */
+    public static class NotNullCacheEntryProcessor implements CacheEntryProcessor<TestKey, TestValue, Object> {
+
```
Comment (SomeFire) [inline modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: No need empty line.

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+        }
+
+        /**
+         * @param val New value.
+         */
+        public void value(String val) {
+            this.val = val;
+        }
+    }
+}
```
Comment (SomeFire) [inline modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: Need empty line in the end of file.

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java [hunk 4]
```diff
@@ -0,0 +1,108 @@
+package org.apache.ignite.cache;
```
Comment (xtern) [inline modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: Apache license header is absent.

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+            try (Transaction transaction = ignite.transactions().txStart(TransactionConcurrency.OPTIMISTIC,
+                TransactionIsolation.SERIALIZABLE, 20000L, 1)) {
+
+                cache.invoke(key, cacheEntryProcessor);
+                cache.invoke(key, cacheEntryProcessor);
+
+                transaction.commit();
+            }
+        }
+        finally {
```
Comment (xtern) [inline modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: I think that better to use try-with-resources
try (Ignite node = startGrid(0)) {
without explicit finally statement.

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java (hunk 3 above)
Comment (voipp) [reply modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: hmm....

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java (hunk 4 above)
Comment (voipp) [reply modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: fixed

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java (hunk 1 above)
Comment (voipp) [reply modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: fixed

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java (hunk 5 above)
Comment (voipp) [reply modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: fixed

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java (hunk 2 above)
Comment (voipp) [reply modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: fixed

File: modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java (hunk 3 above)
Comment (SomeFire) [reply modules/core/src/test/java/org/apache/ignite/cache/IgniteCacheEntryProcessorSequentialCallTest.java]: See point A. https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-A.ConfigureIntelliJIDEAcodestyle

**Pull request closed without merging**